src/System.Runtime.Extensions/tests/System/Net/WebUtility.cs

```diff
@@ -400,23 +436,10 @@ public static void UrlEncodeToBytes_ExcludeIrrelevantData(string value, int offs
             string actual = Encoding.UTF8.GetString(encoded);
             Assert.Equal(expected, actual);
         }
-
-        [Theory]
-        [InlineData("FooBarQuux", 3, 7, "BarQuux")]
-        public static void UrlDecodeToBytes_ExcludeIrrelevantData(string value, int offset, int count, string expected)
```

Comment:
This test here: https://github.com/dotnet/corefx/pull/12123/files/23ce241ab72d3a276f7bc9c40b0be5eaf6b0688b#diff-a9595edda96423a24d85073fdb30a6f4R295
is an example of where no encoding is needed, and we have a custom offset and count